gpu/kinfu/tools/record_tsdfvolume.cpp

```diff
@@ -77,8 +76,8 @@ class DeviceVolume
 {
 public:
 
-  using Ptr = shared_ptr<DeviceVolume>;
-  using ConstPtr = shared_ptr<const DeviceVolume>;
+  using Ptr = std::shared_ptr<DeviceVolume>;
```

Comment:
Keep the shared_ptr unqualified inside pcl namespace (since you're changing this in other places to pcl's shared_ptr